introductory/src/listComponent.jsx

```diff
@@ -0,0 +1,5 @@
+export default function List(props){
+    return (
+        <li>{props.element}</li>
+    )
+}
```

Comment:
Please use a consistent code style for exporting, either by default in-line or at the end of the line